src/__tests__/tv.test.ts

```diff
@@ -2320,7 +2362,7 @@ describe("Tailwind Variants (TV) - Extends", () => {
         {
           isBig: true,
           color: "red",
-          class: "bg-red-500",
+          class: ["bg-red-500"],
```

Comment:
Why the change to the existing test? In general we should avoid changing existing tests.

Reply:
I was just checking if the `compoundVariants` can also be used with array, but i can create new tests for this as well.

Reply:
This could be included in the one new test you added.